clang-tools-extra/clang-doc/HTMLGenerator.cpp

```diff
@@ -516,11 +516,11 @@ writeFileDefinition(const Location &L,
       std::make_unique<TagNode>(HTMLTag::TAG_A, std::to_string(L.LineNumber));
   // The links to a specific line in the source code use the github /
   // googlesource notation so it won't work for all hosting pages.
-  // FIXME: we probably should have a configuration setting for line number
-  // rendering in the HTML. For example, GitHub uses #L22, while googlesource
-  // uses #22 for line numbers.
   LocNumberNode->Attributes.emplace_back(
-      "href", (FileURL + "#" + std::to_string(L.LineNumber)).str());
+      "href",
+      (FileURL + formatv("#{0}{1}", CDCtx.RepositoryLinePrefix.value_or(""),
+                         L.LineNumber))
```

Comment:
Why isn't the `FileURL` part of the format string? you don't need to concatenate there.